### ISSUE TITLE
Accessibility is missing features on older Android APIs

### ISSUE THREAD

**Issue by mklim** [flutter/flutter#29019]
Some of the accessibility APIs aren't available before 18. We could potentially bring in the support library versions of the APIs to fix this.

Right now what's known not to work is:

- [ ] Text editing (`SET_SELECTION`, `COPY`, `CUT`, `PASTE`)
- [ ] ["In list" and "out of list" announcements](https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L312-L316).

**Comment by mklim** [comment]
From @jonahwilliams - we should double check that these APIs are really practically usable in TalkBack on older versions before bringing in support libraries, though. These code paths code have been getting hit without them really being functional even if they were at parity.

**Comment by goderbauer** [comment]
Yeah, last time I tried to bring them in for a11y stuff it increased our binary size significantly. Back then, one idea was to enable ProGuard by default for Flutter to trim out all the stuff from the support libs that we don't use.

However, before we do that, it does make sense to check that old Androids actually do something sensible here.

**Comment by camsim99** [comment]
We are dropping support for Android SDK 16-18, and thus, will not address this issue: https://docs.google.com/document/d/1wWNly2SZRDqupSsHkBWDI_lS2MohBGEwPSXF6W05NOc/edit?usp=sharing
